Makefile

```diff
@@ -220,8 +219,9 @@ bundle-update: verify-previous-version ## Update CSV fields and validate the bun
 	$(MAKE) bundle-validate
 
 .PHONY: verify-previous-version
-verify-previous-version: ## Verifies that PREVIOUS_VERSION variable is set
-	@if [ $(VERSION) != $(DEFAULT_VERSION) ] && [ $(VERSION) != $(CI_VERSION) ] && [ $(PREVIOUS_VERSION) = $(DEFAULT_VERSION) ]; then \
+verify-previous-version: ## Verifies that PREVIOUS_VERSION variable is set properly (but not in CI)
+	@if [[ ($(PREVIOUS_VERSION) != $(DEFAULT_VERSION) && $(PREVIOUS_VERSION) == $(VERSION)) ||\
```

Comment:
when you skip verification of the PREVIOUS_VERISION, you still add the replaces field. With the DEFAULT_VERSION 0.0.1. Does that make sense?

Reply:
Yes, I was oblivious to that.
When running the [bundle-update](https://github.com/medik8s/node-maintenance-operator/blob/main/Makefile#L215) on CI then `verify-previous-version` won't "fail", so the next "steps" in _bundle-update_ are proceeded (including setting the _replaces_ field).